src/internal/operators/skipUntil.ts

```diff
@@ -26,8 +27,12 @@ class SkipUntilOperator<T> implements Operator<T, T> {
   constructor(private notifier: Observable<any>) {
   }
 
-  call(subscriber: Subscriber<T>, source: any): TeardownLogic {
-    return source.subscribe(new SkipUntilSubscriber(subscriber, this.notifier));
+  call(destination: Subscriber<T>, source: any): TeardownLogic {
+    const { notifier } = this;
+    const innerSubscription = new Subscription();
```

Comment:
This looks a little strange, to me. Is there a particular reason the `innerSubscription` cannot be created in the `SkipUntilSubscriber` constructor?